brainstormforce/astra
# AST-550 - Double border comes in customizer preview for above footer

### Description

- When we set top border to the above footer and then change settings for inner element layout then there is border for above footer comes double.

- Steps:
1. Add some components in above footer
2. Add top border
3. Then change the setting for Inner element layout 
4. You can see the issue as shown in the screenshot above.

- JIRA - https://brainstormforce.atlassian.net/browse/AST-550

### Screenshots

As shown in the screenshot - https://share.bsf.io/Z4uKJ55E

### Types of changes


Fix: Double border comes in customizer preview for above footer




### How has this been tested?

- Add widgets / elements in above footer.
- Set Top Border Width option from Design Tab.
- Toggle Inner Container Layout option to either “Stack“ or “Inline“.
- Double border gets applied to above footer in preview, which should not be preset.

### Checklist:
